### PR TITLE
Bedrock <-> Bedrock skin display fix

### PR DESCRIPTION
**Pull request by rtm516**
This needs testing with SkinRestorer but should fix bedrock skin display issues.

### CODE REVIEW

**Comment by bundabrg** [comment]
Works for me

File: connector/src/main/java/org/geysermc/connector/utils/SkinProvider.java [hunk 1]
```diff
@@ -157,10 +158,20 @@ public static Cape getCachedCape(String capeUrl) {
     public static CompletableFuture<SkinAndCape> requestSkinAndCape(UUID playerId, String skinUrl, String capeUrl) {
         return CompletableFuture.supplyAsync(() -> {
             long time = System.currentTimeMillis();
+            String newSkinUrl = skinUrl;
+
+            if (skinUrl == "steve" || skinUrl == "alex") {
```
Comment (jordieh) [inline connector/src/main/java/org/geysermc/connector/utils/SkinProvider.java:163]: Shouldn’t it be equals in case the strings aren’t interned?

**Comment by jennasisis** [comment]
What exactly does this fix? Custom skins and/or skins from the marketplace?

**Comment by DoctorMacc** [comment]
This will fix Bedrock skins displaying for Bedrock clients.

**Review by RednedEpic — APPROVED** [review]

**Pull request merged**